scripts/download_imod_collector.py

```diff
@@ -0,0 +1,73 @@
+import os
+import sys
+import xml.etree.ElementTree as ET
+import zipfile
+from pathlib import Path
+
+import requests
+from tqdm import tqdm
+
+
+def download_imod_collector(tag: str | None) -> None:
+    build_id, build_number = __get_build_info(tag)
+    folder_name = (tag or "develop") + f"_{build_number}"
+    target_folder = Path(".pixi/imod_collector") / folder_name
+
+    if target_folder.exists():
+        print(
+            f"iMOD collector already downloaded at '{target_folder}', remove the folder if you want to enforce re-downloading."
+        )
+        return
+
+    token = os.environ["TEAMCITY_TOKEN"]
+    response = requests.get(
+        f"https://dpcbuild.deltares.nl/app/rest/builds/{build_id}/artifacts/content/imod_coupler_windows.zip",
+        headers={"Authorization": f"Bearer {token}"},
+        stream=True,
+    )
+    response.raise_for_status()
+
+    zip_path = Path(".pixi/imod_coupler_windows.zip")
+    __download_to_file(response, zip_path)
+    __unzip_to_target(target_folder, zip_path)
+
+    os.remove(zip_path)
+
+
+def __get_build_info(tag: str | None) -> tuple[str, str]:
```

Comment:
Why a `__` prefix instead of `_`.
I am not aware of that convention.

Reply:
I always learned `__` is private and a single `_` is protected. But I will change it, no problem.